README.md

```diff
@@ -23,6 +23,23 @@ Get a copy of the app code using the command line:
 - Run the npm instalation on the cvoeo-app project `$ npm install` 
 - Confirm expo can run the project by typing `$ expo start`
 
+### Setup Firebase
+You need to create a free Firebase project at Google.
```

Comment:
Nice, thanks for jumping in on the project!

Could you elaborate a little more on when and why someone would need to create their own firebase project? Right now we haven't been doing this when we bring new people onto the project because we're considering the current instance the development version. 

In fact, right now someone should be able to just pull the repo, run expo, and log in with the test@example.com user. John committed the conf file to git which feels a little odd to me, but takes away yet one more barrier to getting people up and running.